src/main.c

```diff
@@ -127,14 +130,18 @@ int main(int argc,char **argv) {
             {0,0,0,0}
         };
 
-    const char* shortopt = "hc:n:i:j:m:s:p:u:w:s:";
+    const char* shortopt = "hr:c:n:i:j:m:s:p:u:w:s:";
```

Comment:
This was on me, now :sweat: 
`r` doesn't take arguments either... `"hrc:n:i:j:m:s:p:u:w:s:"` is the correct one

Sorry for all the back and forth